numpy/__init__.py

```diff
@@ -332,7 +332,7 @@ def __dir__():
     def _sanity_check():
         """
         Quick sanity checks for common bugs caused by environment.
-        There are some cases e.g. with wrong BLAS ABI that cause wrong
+        There are some cases e.g. with wrong BLAS API that cause wrong
```

Comment:
I think the problem is that a BLAS can have the wrong Application Binary Interface, meaning it was built with the wrong compiler or toolchain, for the platform it is running on.  Wrong API means wrong functions exposed which is pretty rare.

Reply:
https://en.wikipedia.org/wiki/Application_binary_interface